testsuite/performance/tests/src/main/java/org/keycloak/performance/dataset/DatasetLoader.java

```diff
@@ -49,6 +51,7 @@ public DatasetLoader(Dataset dataset, boolean delete) {
         Validate.notNull(dataset);
         this.dataset = dataset;
         this.delete = delete;
+        logger().info(String.format("Opening %s admin clients.", TestConfig.numOfWorkers));
```

Comment:
`logger().info("Opening %d admin clients.", TestConfig.numOfWorkers));`

is probably a bit easier version of this?